### ISSUE TITLE
Setup test server 

### ISSUE THREAD

**Issue by LukasKalbertodt** [elan-ev/tobira#50]
We want to have a test server for a couple of different purposes:

- An Opencast instance with test data. We could of course use `develop.opencast.org` or `stable.opencast.org`, but those are wiped every 24h. Instead, I think it would be useful to have a server where we can have a nice set of test videos and series. That is *not* reset. Additionally, we can roll out custom patches on that server without having to wait for a PR to get merged into Opencast's `develop`.

- A "showcase deployment" of Tobira. This would be a deployment of `master` (or maybe a special branch if we want to deploy there deliberately). This deployment should work most of the time and should be available via a nice-ish URL.

- PR deployments: we want to automatically deploy PRs to make it very easy to test them. As Tobira requires only very little memory, it should be possible to run many instances in parallel on a server. We can expose them via different ports or only use different ports internally and have an nginx forward `server/some-path-to-pr` to a specific port.

**Comment by lkiesow** [comment]
- Opencast test servers now have fixed IDs for test media
    - should suffice for now
    - Maybe dedicated Opencast later if we need anything special
- `tobira.opencast.org` will be the main test deployment
- `<id>.tobira.opencast.org` will be pull request deployments
    - We want to spin up a deployment once we submit or update a pull request
    - We want to drop a deployment once the pull request is closed or merged

**Comment by LukasKalbertodt** [comment]
> Maybe dedicated Opencast later if we need anything special

Once we start working on the Tobira API on the OC side, we certainly want a dedicated OC for us, I'd say.

> We want to drop a deployment once the pull request is closed or merged

Immediately? For Studio, I sometimes used test deployments of already closed PRs. Though rather rarely. Should be fine.

**Comment by lkiesow** [comment]
Fixed with #119
